deb/publish/publish.sh

```diff
@@ -36,11 +36,11 @@ cat $D/merged/Contents | gzip -9c > $D/binary/Contents.gz
 apt-ftparchive -c $bin/release.conf release $D/binary > $D/binary/Release
 # sign the release file
 rm $D/binary/Release.gpg || true
-gpg --batch --no-use-agent --no-default-keyring --keyring $GPG_KEYRING --secret-keyring=$GPG_SECRET_KEYRING --passphrase-file $GPG_PASSPHRASE_FILE \
+gpg --batch --no-use-agent --no-default-keyring --keyring "$GPG_KEYRING" --secret-keyring="$GPG_SECRET_KEYRING" --passphrase-file "$GPG_PASSPHRASE_FILE" \
```

Comment:
Shouldn't it be `"--secret-keyring=$GPG_SECRET_KEYRING"` i.e. quotes around the whole parameter? otherwise you'd just get the quotes as part of the parameter value?